### PR TITLE
AT_01.13.01 | Register button is clickble

### PR DESCRIPTION
**Pull request by AlexMokeev**
https://trello.com/c/5YXzzW0Z/913-tc011301-start-page-register-functionality-register-button-is-clickble

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/// <reference types="Cypress" />
+
+import { StartPage } from "../../../pageObjects/StartPage.js";
+import { RegisterPopup } from "../../../pageObjects/StartPage.js";
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js:4]: please move to 1 import
import { StartPage, RegisterPopup } from "../../../pageObjects/StartPage.js";

File: cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const startPage = new StartPage();
+const registerPopup = new RegisterPopup();
+const randomName = faker.name.firstName();
+const randomCompanyName = faker.company.name();
+const randomEmail = faker.internet.email();
+const randomPhoneNumber = faker.phone.number('+66##########')
+
+
+describe('US_01.13.01 | Register functionality', () => {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js:15]: Please rename describe: US_01.13.01 => US_01.13

File: cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    before(() => {
+        cy.visit('/')
+        startPage.clickRegisterAccountLink()
+    });
+
+    beforeEach(function () {
+        cy.fixture('startPage').then(startPage => {
+            this.startPage = startPage;
+        });
+	});
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js:26]: please move 1 tab to the left

File: cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js (hunk 1 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js:4]: changed

File: cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js (hunk 2 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js:15]: changed

File: cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js (hunk 3 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/startPage/US_01.13_RegisterFunctionality.cy.js:26]: moved

**Review by Rom-T — APPROVED** [review]

**Pull request merged**